### DISCUSSION TITLE
Is it possible for y axis claim not to reach sky ?

### DISCUSSION THREAD

**Discussion by Sleinous** [GriefPrevention/GriefPrevention#1262] [Q&A]
Simple question - possible to deactivate the claim to max build height in GriefPrevention ?
My y axis coords are correctly stated in db for each claim but blocks above those claims are still protected and we'd like to know if it's possible to deactivate that behaviour
Thanks

**Comment by RoboMWM** [comment]
claims in GP are 2D.

May have something for this in v20's API, but no promises.

**Reply by Sleinous** [reply]
Ok cool ! Thanks for the reply :)
